### PR TITLE
Add Target for net7

### PR DESCRIPTION
**Pull request by Iguins**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Bugfix
- Feature
- Code style update (formatting)
- Refactoring (no functional changes, no api changes)
- Build or CI related changes
- Documentation content changes
- Other... Please describe:

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] Interface members are XML documented
- [ ] Documentation (XML or comments) has been added and/or existing documentation has been updated
- [ ] [Architecture documents](./doc/Architecture.md) have been updated
- [ ] Tested on all relevant platforms

### CODE REVIEW

File: src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj [hunk 1]
```diff
@@ -163,6 +163,7 @@
 			<Choose>
 				<When Condition="'$(Configuration)'=='Debug'">
 					<PropertyGroup>
+						<ProvisioningType>manual</ProvisioningType>
```
Comment (Soap-141) [inline src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:166]: Is this required?

**Review by Soap-141 — COMMENTED** [review]
You should also update this `Mono.AotProfiler.Android` I think the version we have is for .NET 6

**Comment by Iguins** [comment]
> You should also update this `Mono.AotProfiler.Android` I think the version we have is for .NET 6

gotcha on it!

File: src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj (hunk 1 above)
Comment (Soap-141) [reply src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:166]: All right!

**Review by Soap-141 — APPROVED** [review]

**Review by Soap-141 — CHANGES_REQUESTED** [review]
You will also have to update the pipeline files, every net6.0 ref and change the sdk version to 7.0.203 which is latest stable

**Comment by Iguins** [comment]
> You will also have to update the pipeline files, every net6.0 ref and change the sdk version to 7.0.203 which is latest stable

Yes It's something I'm doing on a second set of change to do not mix them

**Comment by Soap-141** [comment]
> > You will also have to update the pipeline files, every net6.0 ref and change the sdk version to 7.0.203 which is latest stable
> 
> 
> 
> Yes It's something I'm doing on a second set of change to do not mix them

Ok well you will have to those changes first because the pipeline will never succeed here I feel like.

**Review by Soap-141 — APPROVED** [review]

**Pull request merged**